vaadin-ordered-layout-flow/src/main/java/com/vaadin/flow/component/orderedlayout/FlexLayout.java

```diff
@@ -98,14 +98,28 @@ public FlexLayout(Component... children) {
     }
 
     /**
+     * @deprecated Rename setWrapMode to setFlexWrap
```

Comment:
Please, change this to something like:
```suggestion
     * @deprecated Use {@link #setFlexWrap(WrapMode)} instead
```

(confirm if `@link` is right, first)

Reply:
after using @link, that link is pointing to the right methods. Thank you for comments

Reply:
changed!